### DISCUSSION TITLE
access denied error 5

### DISCUSSION THREAD

**Discussion by mundirkaka0** [sandboxie-plus/Sandboxie#3311] [General]
I downloaded Sandboxie-Classic-x64-v5.66.2.exe ,tried to start steam, but an error came out. Does anyone know how to solve the error?

![SbieCtrl_W87IxdkLjl](https://github.com/sandboxie-plus/Sandboxie/assets/109949472/eeab9f86-e5c2-45b1-b8a5-f197d1061c5a)

**Comment by offhub** [comment]
If file encryption is set for the Steam folder, this may cause an access error. Encrypted files/folders will appear green in Windows Explorer.

**Reply by mundirkaka0** [reply]
what do you mean by encrypted files?

**Reply by bastik-1001** [reply]
If you right-click on a file or folder, then open its properties, click on advanced, you are able to select "Encrypt contents to secure data". If this option is set, the Windows Explorer will display them differently.
